MAUI/DataForm/CustomDataFormItem.md

```diff
@@ -0,0 +1,176 @@
+    <ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+                xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+                xmlns:dataForm="clr-namespace:Syncfusion.Maui.DataForm;assembly=Syncfusion.Maui.DataForm"
+                x:Class="GettingStarted.MainPage">
+        <ContentPage.BindingContext>
+            <local:ViewModel x:Name="bindingContext"/>
+        </ContentPage.BindingContext>
+
+        <ContentPage.Content>
+            <dataForm:SfDataForm x:Name="dataForm" DataObject="{Binding Details}"  AutoGenerateItems="false"> 
+                <dataForm:SfDataForm.Items> 
+                    <dataForm:DataFormTextItem FieldName="Name" /> 
+                    <dataForm:DataFormTextItem FieldName="Password" /> 
+                    <dataForm:DataFormTextItem FieldName="Address" /> 
+                    <dataForm:DataFormAutoCompleteItem FieldName="Countries"  ItemsSource = "{Binding CountryNames}"/> 
+                    <dataForm:DataFormDateItem FieldName="BirthTime" /> 
+                </dataForm:SfDataForm.Items> 
+            </dataForm:SfDataForm>
+        </ContentPage.Content>
+</ContentPage>
+{% endhighlight %}
+{% highlight MainPage.xaml.cs %}
+
+    ObservableCollection<DataFormViewItem> items = new ObservableCollection<DataFormViewItem>();
+    items.Add(new DataFormTextItem() { FieldName = "Name" });
+    items.Add(new DataFormPasswordItem() { FieldName = "Password"});
+    items.Add(new DataFormMultilineItem() { FieldName = "Address" });
+    items.Add(new DataFormAutoCompleteItem() { FieldName = "Countries", ItemsSource = this.GetItemSource("Countries")});
+    items.Add(new DataFormTimeItem() { FieldName = "BirthTime"}); 
+    dataForm.AutoGenerateItems = false; 
+    dataForm.Items = items; 
+
+    public class ViewModel
+    {
+        private DataFormModel details;
+        private IList<string> countryNames;
+        public ViewModel()
+        {
+            details = new DataFormModel();
+            countryNames = new List<string>
+            {
+                "United states",
+                "United Kingdom",
+                "France",
+                "Belgium",
+                "Germany"
+            };
+        }
+
+        public DataFormModel Details
+        {
+            get { return this.details; }
+            set { this.details = value; }
+        }
+        public IList<string> CountryNames
+        {
+            get { return this.countryNames; }
+            set { this.countryNames = value; }
+        }
+    }
+
+    private IList GetItemSource(string sourceName) 
+    { 
+        var list = new List<string>(); 
+        if (sourceName == "Countries") 
+        { 
+            list.Add("India");
+            list.Add("Japan");
+            list.Add("China");
+            list.Add("USA");
+            list.Add("America");
+            list.Add("UK");
+        } 
+        return list; 
+    }
+{% endhighlight %}
+{% endtabs %}
+
```

Comment:
Add topic , 
## Adding DataForm group manually, 
Prepare model, view model for below items

```
Xaml
 <dataForm:DataFormGroupItem Name="Address">
                    <dataForm:DataFormGroupItem.Items>
                        <dataForm:DataFormMultilineItem FieldName="Address"/>
                        <dataForm:DataFormTextItem FieldName="State"/>
                        <dataForm:DataFormTextItem FieldName="ZipCode"/>
                    </dataForm:DataFormGroupItem.Items>

                </dataForm:DataFormGroupItem>
```

```
C#
DataFormGroupItem groupItem = new DataFormGroupItem();
        groupItem.Name = "Address";
        groupItem.Items.Add(new DataFormMultilineItem { FieldName = "Address" });
        groupItem.Items.Add(new DataFormMultilineItem { FieldName = "State" });
        groupItem.Items.Add(new DataFormMultilineItem { FieldName = "ZipCode" });
        this.dataForm.Items.Add(groupItem);
        ```